modules/gaia/scripts/filtering.lua

```diff
@@ -13,11 +13,6 @@ openspace.gaia.documentation = {
         Name = "addClippingSphere",
         Arguments = { name = "String", radius = "Number" },
         Documentation = "Creates a clipping sphere for the Gaia renderable in the first argument"
-    },
-    {
```

Comment:
Is removing this correct?  What was the reason to remove the function?

Reply:
It is a duplicate of the same function defined twice in the documentation section

Reply:
Ah!